test.tsx

```diff
@@ -182,17 +182,47 @@ describe('react2angular', () => {
       expect(baz).toHaveBeenCalledWith(42)
     })
 
-    // TODO: support children
-    it('should not support children', () => {
+    it('should support multiple children', () => {
       const scope = Object.assign($rootScope.$new(true), {
         bar: [true, false],
         baz: (value: number) => value + 1,
         foo: 1
       })
-      const element = $(`<test-angular-one foo="foo" bar="bar" baz="baz"><span>Transcluded</span></test-angular-one>`)
+      const element = $(`<test-angular-one foo="foo" bar="bar" baz="baz"><h1>transcluded</h1><h2>transcluded</h2></test-angular-one>`)
       $compile(element)(scope)
       $rootScope.$apply()
-      expect(element.find('span').length).toBe(0)
+      expect(element.find('h1').length).toBe(1)
+      expect(element.find('h1').text()).toBe('transcluded')
+      expect(element.find('h2').length).toBe(1)
+      expect(element.find('h2').text()).toBe('transcluded')
+    })
+
+    it('should support transclusion of other components', () => {
+      const scope = Object.assign($rootScope.$new(true), {
+        bar: [true, false],
+        baz: (value: number) => value + 1,
+        foo: 1
+      })
+      const element = $(`<test-angular-one foo="foo" bar="bar"><test-angular-two foo="foo" bar="bar"></test-angular-two></test-angular-one>`)
+      $compile(element)(scope)
+      $rootScope.$apply()
+      expect(element.find('div').length).toBe(2)
+      expect(element.find('p').length).toBe(6)
+    })
+
+    it('should update with children', () => {
+      const scope = Object.assign($rootScope.$new(true), {
+        bar: [true, false],
+        foo: 1
+      })
+      const element = $(`<test-angular-one foo="foo" bar="bar"><span>{{foo}}</span></test-angular-one>`)
+      $compile(element)(scope)
+      $rootScope.$apply()
+      expect(element.find('span').text()).toBe('1')
+      // scope.$apply(() =>
```

Comment:
@bcherny we can now transclude other components as well, and bind once with initial rendering. However, I cannot manage to update the two way binding. Any idea? Would it be possible to release without this support for now?

Reply:
@spirosikmd This looks like a pretty serious bug to me. In your test it's not that the view doesn't update, it's that it reverts to its original untemplated version (eg. in this case it's `{'{'}{'{'}foo{'}'}{'}'}`. If that's correct, I would argue this should be fixed before we merge. Broken support for transclusion (that sets false expectations) could be worse than no support.

Reply:
You are definitely right. This should be fixed first!